tc/sgx/trusted_worker_manager/enclave/work_order_preprocessed_keys_wpe.cpp

```diff
@@ -37,6 +37,14 @@ void WorkOrderPreProcessedKeys::Unpack(const JSON_Object* keys_object,
 
     data_keys.index = GetJsonNumber(keys_object, "index");
     encrypted_data_key = GetJsonStr(keys_object, "encrypted-data-key");
+
+    // skip decrypting data encryption key if its value is
+    // equivalent to empty or null or "-"
+    if (encrypted_data_key.empty() ||  "null" == encrypted_data_key ||
+        "-" == encrypted_data_key) {
+        return;
```

Comment:
Does `data_keys.decrypted_data` has some valid data at this point. Or should it be explicitly set it as  `data_keys.decrypted_data = Base64EncodedStringToByteArray(encrypted_data_key)`

Reply:
At this point, data_keys.decrypted_key will have valid data in ByteArray form. No Base64 conversion is needed/